rust/src/history.rs

```diff
@@ -211,7 +213,7 @@ fn history_prune_inner() -> Result<()> {
 }
 
 pub(crate) fn history_prune() -> CxxResult<()> {
-    Ok(history_prune_inner().context("Failed to prune history")?)
+    Ok(history_prune_inner()?)
```

Comment:
I think the only reason for this function was for error-wrapping the inner call, so we should be able to drop it entirely now.